src/Symfony/Component/Form/Tests/Extension/Validator/ValidatorExtensionTest.php

```diff
@@ -38,9 +38,11 @@ public function test2Dot5ValidationApi()
             ->method('addPropertyConstraint')
             ->with('children', $this->isInstanceOf('Symfony\Component\Validator\Constraints\Valid'));
 
-        $validator
-            ->expects($this->never())
-            ->method('getMetadataFactory');
+        if (method_exists($validator, 'getMetadataFactory')) {
```

Comment:
I would rather check whether whether it implements the legacy interface at the same time, to make it easier to understand the test